draft-trammell-taps-interface.md

```diff
@@ -1306,6 +1309,8 @@ type Connection interface {
 }
 
 type Content interface {
+\[MICHAEL: We need something here for metadata, for a receiver -
+at least ECN must be a part of this metadata.]
```

Comment:
yeah, I thought about adding a simple Get(k string) interface{} metadata accessor, but we'd need to add metadata language to the receive section. I can do this after this PR and preconnection land...